src/FakePdoTrait.php

```diff
@@ -139,6 +139,11 @@ public function rollback()
         return true;
     }
 
+    public function inTransaction()
+    {
+        return Server::haveSnapshot('transaction');
```

Comment:
sorry for nitpicking: should it be `hasSnapshot` instead?

Reply:
yes, that's better English for sure